experimental/generation/generator/packages/library/src/dialogGenerator.ts

```diff
@@ -232,7 +232,12 @@ function addPrefixToImports(template: string, scope: any): string {
 }
 
 function addPrefix(prefix: string, name: string): string {
-    return `${prefix}-${name}`
+    let dir = name.lastIndexOf('/')
```

Comment:
Is name ever null or undefined?

Reply:
No

---
In reply to: [491180855](https://github.com/microsoft/BotBuilder-Samples/pull/2750#discussion_r491180855) [](ancestors = 491180855)